### ISSUE TITLE
"Digital Inputs" randomly going from "CLOSED" to "NOT_PRESSED"

### ISSUE THREAD

**Issue by VNDCK** [ridiekel/jeletask#33]
Hello! Since the new update, i've added some buttons. However, I don't use short/long press - but only "press". When I press a button, I get the following changes.

- Button changed to CLOSED
- Button changed to NOT_PRESSED

```
2022-11-18 15:09:07.600 -DEBUG - i.g.r.j.mqtt.listener.MqttProcessor      : [EVENT  ] - [INPUT     ] - [ 29] - [          Bureau - Drukknop rechts] - publishing topic 'teletask/overes/input/29/state' -> {"state":"CLOSED"}
2022-11-18 15:09:07.600 - INFO - i.g.r.j.mqtt.listener.MqttProcessor      : [EVENT  ] - [INPUT     ] - [ 29] - [          Bureau - Drukknop rechts] - {"state":"CLOSED"}
2022-11-18 15:09:08.101 -DEBUG - i.g.r.j.mqtt.listener.MqttProcessor      : [EVENT  ] - [INPUT     ] - [ 29] - [          Bureau - Drukknop rechts] - publishing topic 'teletask/overes/input/29/state' -> {"state":"NOT_PRESSED"}
2022-11-18 15:09:08.101 - INFO - i.g.r.j.mqtt.listener.MqttProcessor      : [EVENT  ] - [INPUT     ] - [ 29] - [          Bureau - Drukknop rechts] - {"state":"NOT_PRESSED"}
2022-11-18 15:09:35.639 -TRACE - i.g.r.j.c.b.m.messages.MessageSupport    : Sending message: {}
2022-11-18 15:09:35.639 -DEBUG - i.g.r.j.client.TeletaskClientImpl       
```

Sadly, I can't use this in automations/scripts, because these events seem to happen at random as well. 

- Is this expected?
- Is it possible to change to config of a short press, so all my pressed are detected as a "short press"?

**Comment by VNDCK** [comment]
Hey Guys! Any updates or tips on this one?

thx!

**Comment by ridiekel** [comment]
Hi,

I'm going to look into this, but first I want to have a stable test suite. The problem we have now, is that I am never sure I don't break other stuff when working on this project.

I want to have tests that test the HA integration, so when I refactor or add features, I can be pretty sure everything keeps working.

I'm working on those tests and the framework around that in the features/testcontainers branch.

Once that is finished, we can start looking into fixing these issues.

**Comment by VNDCK** [comment]
Thank you! Let me know if I can help you with anything regarding testing on HA.

**Comment by ridiekel** [comment]
Please test with the latest version and if you still have issues, please create a new issue. I'm trying to clean up some old tickets.
